src/controllers/returns.js

```diff
@@ -0,0 +1,32 @@
+// eslint-disable-next-line unicorn/import-index, import/no-useless-path-segments
+import database from '../models/index.js';
+const {Application, Returns} = database;
+
+/**
+ * An object to perform 'persistence' operations on our Return objects.
+ */
+const ReturnsController = {
+  /**
+   * Create a new randomly allocated Return wrapped in a database transaction.
+   *
+   * Transaction completes all requests and returns the new return transaction id.
+   *
+   * @param {number} id An existing return's ID.
+   * @param {any} cleanObject A new revocation object to be added to the database.
+   * @returns {number} The new Return transaction id.
```

Comment:
Technically this is `{number | undefined}`.

Reply:
resolved https://github.com/Scottish-Natural-Heritage/standard-forestry-operations-api/pull/30/commits/acdef55b0a4c2c8d702acb2c868b79fb148ea555